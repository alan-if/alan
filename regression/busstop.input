examine the car
examine the bus
examine the sign
examine martha
