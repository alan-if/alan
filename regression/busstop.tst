alan busstop
arun busstop < busstop.input
