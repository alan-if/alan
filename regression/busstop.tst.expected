########## busstop ##########
../compiler/alan busstop
../interpreter/arun -n busstop < busstop.input
########## /bin/sh  busstop.tst ##########

        No warnings or errors detected.
        5 informational message(s).



At the Bus Stop
A car drives by. There is a bus and a sign here. Martha is here.

> examine the car
You can not examine the car.

> examine the bus
The bus is red. ... and stopped at the bus stop.

> examine the sign
There is nothing special about it.

> examine martha
Martha resents being scrutinized in that way.

> 

Do you want to RESTART, RESTORE or QUIT ? 
